Editor/MobiledgeXEditorWindow.cs

```diff
@@ -68,6 +68,35 @@ private Texture2D MakeTex(int width, int height, Color col)
         }
 
         private string sdkVersion;
+        private int selectedRegionIndex = 0;
+        private string[] regionOptions = new string[5] { "Nearest", "EU", "JP", "KR", "US" };
+        private string region;
+        public string Region
+        {
+            set { region = value; }
+            get
+            {
+                switch (region)
+                {
+                    case "EU":
+                        return EU_DME;
+                    case "KR":
+                        return KR_DME;
+                    case "JP":
+                        return JP_DME;
+                    case "US":
+                        return US_DME;
+                    case "Nearest":
+                    default:
+                        return "";
+                }
+            }
+        }
+        const string EU_DME = "eu-mexdemo.dme.mobiledgex.net";
+        const string KR_DME = "kr-mexdemo.dme.mobiledgex.net";
+        const string US_DME = "us-mexdemo.dme.mobiledgex.net";
+        const string JP_DME = "jp-mexdemo.dme.mobiledgex.net";
+        const uint DME_PORT = 38001;
```

Comment:
If in namespace, use the MatchingEngine constant, one source of data only, if possible.

Reply:
Good catch, Thanks